documentation/jetty-documentation/src/main/asciidoc/programming-guide/maven/jetty-maven-plugin.adoc

```diff
@@ -57,13 +57,15 @@ We recommend either the traditional link:{DISTGUIDE}[distribution] deployment ap
 [[get-up-and-running]]
 ==== Get Up and Running
 
-First, add `jetty-maven-plugin` to your `pom.xml` definition:
+Since Jetty 12, Jetty Maven plugin is repackaged for corresponding Jakarta EE version with an `eeX` classifier in the groupId and artifactId. The `eeX` can be `ee8`,`ee9` and `ee10`, stands for Jakarta EE 8, 9 and 10.
+
+First, add `jetty-ee{8,9,10}-maven-plugin` to your `pom.xml` definition:
```

Comment:
I did this in the previous version, but @olamy require a general approach to cover all Jakarta EE version.

Reply:
maybe because @janbartel wants everybody use ee10 only :P 
with adding the ` Here's an example of how to do that for EE10`  we should be fine.
when ee11 comes we should not forget to change otherwise people will be stuck with ee10 for life :)